desktop/core/src/desktop/templates/scheduler/submit_job_popup.mako

```diff
@@ -163,7 +164,18 @@ else:
       else if (_el.val().indexOf("T") == "-1") {
         _el.val(_el.val() + "T" + _val.split("T")[1]);
       }
+
+      $(document).off("click.hideDatepicker", function (event) {
+        event.stopPropagation();
+      });
```

Comment:
This has no effect. When a function is provided to off it must be equal to the function provided in the `on` call. To remove all event handlers do `$(document).off("click.hideDatepicker");`